### PR TITLE
feat: added information about package publisher for notifications

### PR DESCRIPTION
**Pull request by ilyaztsv**
the feature for #739

### CODE REVIEW

File: test/functional/notifications/notify.js [hunk 1]
```diff
@@ -1,118 +1,174 @@
-import assert from 'assert';
-import _ from 'lodash';
+import assert from "assert";
```
Comment (juanpicado) [inline test/functional/notifications/notify.js]: you need to check what sort of formatting you are using, many places are affected. In this specific place, we use a single quote ( I wonder why lint is not caching this)

File: docs/notifications.md [hunk 2]
```diff
@@ -38,13 +38,39 @@ notify:
...
-Property | Type | Required | Support | Default | Description
---- | --- | --- | --- | --- | ---
-method| string | No | all |  | HTTP verb
-packagePattern| string | No | all |  | Only run this notification if the package name matches the regular expression
-packagePatternFlags| string | No | all |   | Any flags to be used with the regular expression
-headers| array/object | Yes | all |  | If this endpoint requires specific headers, set them here as an array of key: value objects.
-endpoint| string | Yes | all |  | set the URL endpoint for this call
-content| string | Yes | all |  | any [handlebar](https://handlebarsjs.com/) expressions
+| Property            | Type         | Required | Support | Default | Description                                                                                  |
+| ------------------- | ------------ | -------- | ------- | ------- | -------------------------------------------------------------------------------------------- |
```
Comment (juanpicado) [inline docs/notifications.md]: All those white spaces need to be clean up

File: src/api/endpoint/api/publish.js [hunk 3]
```diff
@@ -17,8 +17,11 @@ export default function(router: Router, auth: IAuth, storage: IStorageHandler, c
   const can = allow(auth);
 
   // publishing a package
-  router.put('/:package/:_rev?/:revision?', can('publish'),
-    media(mime.getType('json')), expectJson, function(req: $RequestExtend, res: $ResponseExtend, next: $NextFunctionVer) {
+  router.put('/:package/:_rev?/:revision?', can('publish'), media(mime.getType('json')), expectJson, function(
```
Comment (juanpicado) [inline src/api/endpoint/api/publish.js]: This is just formatting, if there is no need has to remains the same.

File: test/functional/notifications/notify.js (hunk 1 above)
Comment (ilyaztsv) [reply test/functional/notifications/notify.js]: fixed

File: docs/notifications.md (hunk 2 above)
Comment (ilyaztsv) [reply docs/notifications.md]: fixed

File: src/api/endpoint/api/publish.js (hunk 3 above)
Comment (ilyaztsv) [reply src/api/endpoint/api/publish.js]: fixed all extra formatting changes

**Comment by ilyaztsv** [comment]
@juanpicado 
> Check comments
fixed. please check it :)

**Comment by juanpicado** [comment]
Oh yeah !! much better now ! I'll test it this afternoon :) 

thanks @ilyaztsv

**Comment by ilyaztsv** [comment]
Thanks! 👍

**Comment by juanpicado** [comment]
Thanks to you for the PR ! 👍

**Pull request merged**